grammar.js

```diff
@@ -1056,6 +1071,23 @@ module.exports = grammar({
       ),
     ),
 
+    create_schema: $ => prec.left(seq(
+      $.keyword_create,
+      $.keyword_schema,
+      choice(
+        seq(
+          optional($._if_not_exists),
+          $.identifier,
+          optional(seq($.keyword_authorization, $.identifier)),
+        ),
+        seq(
+          $.keyword_authorization,
+          $.identifier,
+        ),
+      ),
+      repeat($._create_statement),
```

Comment:
structuring this form into `program` (now a `choice`) and deleting the repeat line seems to work well. It does require precedence over _create_statement -> create_schema but I think that's reasonable

```js
      prec(2, seq(
        $.create_schema,
        repeat($._create_statement),
      )),
```

Reply:
Not sure that I understand this correctly. Isn't the `program` node a bit too close to the root of the tree?

I have put it in the `create_statement` with a left prec.